### PR TITLE
Recommend Pressable over Touchable for alternative to Button

### PR DESCRIPTION
**Pull request by NickGerleman**
Fix this sentence with more up to date guidance.

### CODE REVIEW

**Comment by NickGerleman** [comment]
The result of https://js.coach/?menu%5Bcollections%5D=React%20Native&page=1&query=button does not seem to not be helpful, so will remove that link

File: docs/button.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Button
 
 A basic button component that should render nicely on any platform. Supports a minimal level of customization.
 
-If this button doesn't look right for your app, you can build your own button using [TouchableOpacity](touchableopacity) or [TouchableWithoutFeedback](touchablewithoutfeedback). For inspiration, look at the [source code for this button component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js). Or, take a look at the [wide variety of button components built by the community](https://js.coach/?menu%5Bcollections%5D=React%20Native&page=1&query=button).
+If this button doesn't look right for your app, you can build your own button using [Pressable](pressable). For inspiration, look at the [source code for this button component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js). Or, take a look at the [wide variety of button components built by the community](https://js.coach/?menu%5Bcollections%5D=React%20Native&page=1&query=button).
```
Comment (NickGerleman) [inline docs/button.md:8]:
```suggestion
If this button doesn't look right for your app, you can build your own button using [Pressable](pressable). For inspiration, look at the [source code for the Button component](https://github.com/facebook/react-native/blob/main/Libraries/Components/Button.js).

**Comment by NickGerleman** [comment]
Oh whoops it looks like #3415 does the same thing.

**Review by Simek — APPROVED** [review]
LGTM! 👍

**Pull request merged**